docs/src/examples/Lazy-abstraction-reachability.jl

```diff
@@ -100,129 +101,88 @@ function compute_reachable_set(rect::UT.HyperRectangle, contsys, Udom)
     return UT.HyperRectangle(lb, ub)
 end
 
-function get_transitions(symmodel, sys, source)
-    return SY.get_transitions_1(symmodel, sys, source, compute_reachable_set)
-end
+concrete_problem = SimpleProblem.problem()
+concrete_system = concrete_problem.system
 
-function minimum_transition_cost(symmodel, contsys, source, target)
-    return 1.0
-end
+hx = [0.5, 0.5]
+u0 = SVector(0.0, 0.0)
+hu = SVector(0.5, 0.5)
+Ugrid = DO.GridFree(u0, hu)
+maxIter = 100
 
-## problem data
-struct SimpleSystem{N, T, F <: Function} <: ST.ControlSystem{N, T}
-    tstep::Float64
-    measnoise::SVector{N, T}
-    sys_map::F
-end
+optimizer = MOI.instantiate(AB.LazyAbstraction.Optimizer)
 
-function NewSimpleSystem(tstep, measnoise::SVector{N, T}) where {N, T}
-    function sys_map(x::SVector{N, T}, u, tstep)
-        return x + tstep * u
-    end
-    return SimpleSystem(tstep, measnoise, sys_map)
-end
+AB.LazyAbstraction.set_Optimizer!(
```

Comment:
same here, seems weird to me that a function has a capital letter without any initials or else

this should'nt be set_optimizer!(..) ?

Reply:
Same for other solvers..